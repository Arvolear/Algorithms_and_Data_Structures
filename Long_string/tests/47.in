1783693575320689444513139239787877139686131301445
18431557445425150039964294561173916364028781208322