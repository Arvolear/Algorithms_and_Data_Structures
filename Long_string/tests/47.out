20215251020745839484477433800961793503714912509767
16647863870104460595451155321386039224342649906877
32876250598559059188380712523758134761810867430560843585797867011209724600866350384661133524625290
10